src/views/Point.js

```diff
@@ -130,6 +130,8 @@ export default function Point() {
     isOwner,
   } = useCurrentPermissions();
 
+  const canBitcoin = Just.hasInstance(urbitWallet);
```

Comment:
Considering this doesn't depend on the specific point in any way, but rather on the login credentials, it's a bit weird that this displays on the point page, alongside point-specific menus.

imo it should at least _also_ be displayed on the `Points` overview page, and maybe shown as in a "separate" category somehow on this individual point page. cc @urcades, @jyng

Reply:
Yeah didn't think about this at all, the BTC wallet doesn't have anything to do with points at all! Definitely should be on the `Points` page somehow or in a separate category. Added c8e8454 but need to think about this for a bit.